LibertyDSNP/project-liberty-web
# animations added to footer and header logo

Problem
=======
As a user, I want to see animations and improve the styling and feel of the site.
[link to Pivotal Tracker #177295460](https://www.pivotaltracker.com/story/show/177295460)

Solution
========
- figure out the tool that we want to use to integrate the animations
- look to old site to see where animations should be applied
- apply animations

Change summary:
---------------
* Chose [AOS](https://github.com/michalsnik/aos)
* Implemented animations, including a custom animation

Steps to Verify:
----------------
1. Make sure they work
2. check mobile when deployed
